backend/tests/test_config.py

```
@@ -249,6 +249,80 @@ def test_dev_auth_cannot_enable_for_production_or_staging(self) -> None:
+
+    def test_production_identity_bridge_rejects_placeholder_shared_secret(self) -> None:
+        with self.assertRaisesRegex(
+            ValidationError,
+            "SAFEQUERY_PRODUCTION_IDENTITY_BRIDGE_SHARED_SECRET",
+        ):
+            Settings(
+                app_postgres_url="postgresql://safequery:safequery@db:5432/safequery",
+                production_identity_bridge_enabled=True,
+                production_identity_bridge_trusted_issuer="https://idp.example.test",
+                production_identity_bridge_trusted_source="saml-oidc-bridge",
+                production_identity_bridge_shared_secret="change-me",
```

Ruff (0.15.11)

[error] Possible hardcoded password assigned to argument: "production_identity_bridge_shared_secret"

(S106)